code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -31,16 +31,9 @@
 		user << "You [locked ? "lock" : "unlock"] \the [src] access panel."
 		return
 
-	var/list/usable_qualities = list()
-	if(open)
-		usable_qualities.Add(QUALITY_WIRE_CUTTING, QUALITY_PULSING, QUALITY_CUTTING, QUALITY_BOLT_TURNING, QUALITY_SCREW_DRIVING)
-	if(!open && locked)
-		usable_qualities.Add(QUALITY_PRYING)
```

Comment:
Why change this section? 

I think the point of limiting the useable qualities list is to reduce cases where you're unable to do a normal interaction because one of its tool qualities take over.
And particularly to reduce cases where an item has multiple tool qualities and the one that gets first keeps superceding the one you want to use.
We should query as few tool qualities as possible

Reply:
In addition to correcting the wrong condition for when a crowbar can be applied, intent was to give the user feedback about his actions: despite using a tool that is not compatible with the current item state, it will tell them the problem. In this case, every tool except crowbar will tell the user that they need to open the access panel first, and applying the crowbar will tell them that they need to unlock it first. I would think it is preferable to receiving no feedback at all after using the wrong tool.

I will change line 125 because it should say "is locked" instead of "is closed"

Reply:
thats a good reason, okay approving